modules/opengl/src/shader/shader.cpp

```diff
@@ -370,7 +370,9 @@ std::string Shader::processLog(std::string log) const {
     return result.str();
 }
 
-bool Shader::isReady() const {
+bool Shader::isReady() const { return ready_ && checkLinkStatus(); }
```

Comment:
do we have to call checkLinkStatus here? would it not be enough to return ready_? we only make that true if we have built and linked right?

Reply:
is that state reset when a shader is reloaded?

Reply:
think so

Reply:
yes, a file change will trigger a rebuild which starts by setting ready=false

Reply:
By looking at the code it seems we can remove the `&& checkLinkStatus()`, I am testing it now.

Reply:
👍